exasol_advanced_analytics_framework/udf_framework/udf_query_handler.py

```diff
@@ -1,7 +1,10 @@
 from typing import Dict, Any
 
+from exasol_advanced_analytics_framework.query_handler.context.scope_query_handler_context import \
+    ScopeQueryHandlerContext
 from exasol_advanced_analytics_framework.query_handler.query_handler import QueryHandler
 
 
-class UDFQueryHandler(QueryHandler[Dict[str, Any], Dict[str, Any]]):
+class UDFQueryHandler(QueryHandler[str, str]):
     pass
```

Comment:
Learned from Nico that it's better to use a docstring in this case :) 
```suggestion
    """ description """
```